docker-compose.test.yml

```diff
@@ -1,41 +0,0 @@
-version: "2"
```

Comment:
The lib template still has a docker-compose.test.yml file for being able to run a test from a docker container.